lib/config/config-file.js

```diff
@@ -547,6 +548,43 @@ function load(filePath, applyEnvironments, relativeTo) {
     return config;
 }
 
+/**
+ * Loads a config object, applying extends if present.
+ * @param {Object} configObject a config object to load
+ * @returns {Object} the config object with extends applied if present, or the passed config if not
+ * @private
+ */
+function loadObject(configObject) {
+    return configObject.extends ? applyExtends(configObject, "") : configObject;
+}
+
+/**
+ * Loads a config object from the config cache based on its filename, falling back to the disk if the file is not yet
+ * cached.
+ * @param {string} filePath the path to the config file
+ * @returns {Object} the parsed config object (empty object if there was a parse error)
+ * @private
+ */
+function loadCached(filePath) {
+    const cachedConfig = configCache.getConfig(filePath);
+    let config = {};
```

Comment:
Initializing `config` to an empty object seems unnecessary here, because it's reassigned immediately below.